src/main/java/org/cyclonedx/model/vulnerability/Vulnerability.java

```diff
@@ -94,6 +95,10 @@ public Vulnerability() {}
   @JacksonXmlElementWrapper(localName = "tools")
   @JacksonXmlProperty(localName = "tool")
   private List<Tool> tools;
```

Comment:
`tools` should be marked as deprecated to avoid confusion, similar to how it's done in `Metadata`:

https://github.com/CycloneDX/cyclonedx-core-java/blob/3702283abd718595c069025762979fb9dfcee26b/src/main/java/org/cyclonedx/model/Metadata.java#L63-L65

Reply:
> `tools` should be marked as deprecated to avoid confusion, similar to how it's done in `Metadata`:
> 
> https://github.com/CycloneDX/cyclonedx-core-java/blob/3702283abd718595c069025762979fb9dfcee26b/src/main/java/org/cyclonedx/model/Metadata.java#L63-L65

👍 7fc564a0